deployment/docker/uwsgi.conf

```diff
@@ -1,12 +1,13 @@
 [uwsgi]
 
 # Touch this file to reload uwsgi
-#touch-reload = /tmp/touch-me-to-reload
+touch-reload = /tmp/django.pid
 chdir = /home/web/django_project
 module = core.wsgi
 master = true
-pidfile=/tmp/django.pid
+pidfile = /tmp/django.pid
 socket = 0.0.0.0:8080
+http-socket = 0.0.0.0:8000
```

Comment:
Allow http socket in case you want to see the output directly inside the container (for healthcheck too)